bucket/jpegview.json

```diff
@@ -24,6 +25,11 @@
         "url": "https://sourceforge.net/projects/jpegview/rss?path=/",
         "re": "/jpegview/([\\d.]+)/JPEGView_"
     },
+    "persist": [
+        "JPEGView.ini",
+        "KeyMap.txt",
+        "ParamDB.db"
```

Comment:
Where did you get information about this file?

When it is created it is always 0bytes and also when you delete it application do not create it.

Reply:
![A](https://i.imgur.com/7AvgFaa.png)

Reply:
You also need to copy old folder to persist for backward compatability

Reply:
If you just create it then it cannot be used

![A](https://i.imgur.com/Zcfq5dG.png)

See ditto manifest how to handle this